cmd/up.go

```diff
@@ -177,14 +177,12 @@ func (up *UpContext) Activate() {
 			return
 		}
 
-		activated, err := up.devMode(retry, d, create)
+		err = up.devMode(retry, d, create)
 		if err != nil {
 			up.Exit <- err
 			return
 		}
-		if !activated {
-			return
```

Comment:
This return made the process hang, since we have a loop that only exits if there's an error, or context.Done() returns